Pre-processing/PlotAllChannels.py

```diff
@@ -1,21 +1,38 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from mne.io import concatenate_raws, read_raw_edf
+import os
 
 
 #Load Data
-load_path = 'C:/Users/User/Desktop/CSE 481C - Neruo Capstone/Project Repo/Data/TUH/00003010_s003_t000.edf'
+filename = '00003010_s003_t000'
+fileformat = '.edf'
+load_path = 'C:/Users/User/Desktop/CSE 481C - Neruo Capstone/Project Repo/Data/TUH/' + filename + fileformat
```

Comment:
In general relative paths are preferred, but passing in filename should make this a non-issue. When web-hosting this will be a more interesting problem. We should try to have pre-processing code run on the front end